test-jest/cartesian/Brush.spec.tsx

```diff
@@ -22,8 +22,13 @@ describe('<Brush />', () => {
     { date: '2023-01-14', value: 10 },
   ];
 
-  test('Render 2 travellers and 1 slide in simple Brush', () => {
-    const { container } = render(<Brush x={100} y={50} width={400} height={40} data={data} />);
+  test('Render 2 travelers and 1 slide in simple Brush', () => {
+    const { container } = render(
+      <BarChart width={400} height={100} data={data}>
```

Comment:
any reason this is being wrapped in a bar chart?

Reply:
Same as this [comment](https://github.com/recharts/recharts/pull/3269#discussion_r1084847142)